integration-cli/requirements.go

```diff
@@ -25,6 +26,13 @@ var (
 		func() bool { return supportsExec },
 		"Test requires 'docker exec' capabilities on the tested daemon.",
 	}
+	NativeExecDriver = TestRequirement{
+		func() bool {
+			driver := os.Getenv("DOCKER_EXECDRIVER")
```

Comment:
I actually double checked this last night, there is nothing to do about this one.